monitoring/ai-monitor/src/bot/tools.py

```
@@ -0,0 +1,183 @@
+            timeout=self.timeout,
+        )
+        resp.raise_for_status()
+        data = resp.json()
+        result = data.get("data", {}).get("result", [])
+        if not result:
+            return {"count": 0, "values": []}
+        # 첫 시리즈만 (여러 시리즈 반환 시 간결성 위해)
+        values = result[0].get("values", [])
+        # 샘플이 많으면 앞뒤 각 20개만 전달 (Claude 컨텍스트 절약)
+        if len(values) > 40:
+            values = values[:20] + [("...", "truncated")] + values[-20:]
```

Ruff (0.15.10)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)